### PR TITLE
LRQA-66445 | master

### PR DESCRIPTION
**Pull request by Tim-Cao**
https://issues.liferay.com/browse/LRQA-66445

This is only for master.

@david-gutierrez-mesa
@kenjiheigel

### CODE REVIEW

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:echo:nocompile

**Comment by Tim-Cao** [comment]
ci:stop

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:echo:nocompile

File: portal-web/test/functional/com/liferay/portalweb/functions/WaitForPageLoad.function [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@default = "waitForPageLoad"
+definition {
+
+	function waitForPageLoad {
+		var javascript = '''
+			var state = document.readyState;
+
+			return state === "complete";
```
Comment (kenjiheigel) [inline portal-web/test/functional/com/liferay/portalweb/functions/WaitForPageLoad.function:8]: @Tim-Cao looks good, but you can simplify this further: 
```
return document.readyState === "complete";
```

Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/functions/WaitForPageLoad.function:8]: Sure

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/4190

**Comment by Tim-Cao** [comment]
https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1524085730
The same results as previous. The solution looks good enough.

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**